nixos/modules/services/misc/paperless.nix

```
@@ -291,6 +296,31 @@ in
       };
     };
 
+    # Download NLTK corpus data
+    systemd.services.paperless-download-nltk-data = {
+      wantedBy = [ "paperless-scheduler.service" ];
+      before = [ "paperless-scheduler.service" ];
+      after = [ "network-online.target" ];
+      serviceConfig = defaultServiceConfig // {
+        User = cfg.user;
+        Type = "oneshot";
+        # Enable internet access
+        PrivateNetwork = false;
```

Restrict the downloader to only access the cache dir.
```suggestion
        PrivateNetwork = false;
        BindPaths = [];
```